app/views/search/_collection_list_item.html.erb

```diff
@@ -29,7 +29,7 @@
               <p>
                 <span class="panel-list-details" <% if is_project %>data-bind="style: {'background-color': $root.project.colorForCollection($data)}"<% end %>>
                   <span data-bind="text: short_name"></span>
-                  <!-- ko if: $data.version_id && $data.version_id !== 'Undefined' -->
+                  <!-- ko if: $data.version_id && $data.version_id !== 'Undefined' && $data.version_id !== 'Not provided' -->
```

Comment:
Sorry for not being more clear on this, but this is what I was saying we shouldn't do. If it's a metadata problem, we need to kick it back. People are hacking around a metadata requirement. I wonder if anything is 'Undefined' currently.